docs/source/conf.py

```diff
@@ -0,0 +1,132 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+import datetime
+
+project = 'PyDAGMC'
+copyright = '2023 - {}, The UW CNERG Team. All rights reserved'.format(datetime.date.today().year)
+author = 'Patrick Shriwise, Paul Wilson, et al'
```

Comment:
There are other authors who deserve more recognition than me (e.g. Romano, Peterson).  We should either be comprehensive or find another way to list authors here.